blueocean-personalization/src/main/js/components/FavoritePipeline.jsx

```diff
@@ -0,0 +1,91 @@
+/**
+ * Created by cmeyers on 7/8/16.
+ */
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { createSelector } from 'reselect';
+import { List } from 'immutable';
+
+import { Favorite } from '@jenkins-cd/design-language';
+
+import { favoritesSelector } from '../redux/FavoritesStore';
+import { actions } from '../redux/FavoritesActions';
+import { checkMatchingFavoriteUrls } from '../util/FavoriteUtils';
+
+/**
+ */
```

Comment:
Hey Cliff .... could you add another line or two to explain what this is use for etc.